tests/test_wave_05.py

```diff
@@ -113,18 +128,15 @@ def test_delete_goal(client, one_goal):
     response = client.get("/goals/1")
     assert response.status_code == 404
 
-@pytest.mark.skip(reason="test to be completed by student")
 def test_delete_goal_not_found(client):
-    pass
-
     # Act
-    # ---- Complete Act Here ----
+    response = client.delete("/goals/1")
+    response_body = response.get_json()
 
     # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # ---- Complete Assertions Here ----
+    assert response.status_code == 404
+    assert response_body == None
+    assert Goal.query.all() == []
 
```

Comment:
It might also be good to have tests for goals that have tasks and goals that do not have tasks.